src/modules/governance/DelegatedInfoPanel.tsx

```diff
@@ -64,7 +64,7 @@ const DelegatedPower: React.FC<DelegatedPowerProps> = ({
           </Row>
         ) : (
           <>
-            {!isAaveSelfDelegated && (
+            {!isAaveSelfDelegated && aavePower !== '0' && (
```

Comment:
@0x4Graham can we remove the delegation stuff if its not relevant

Reply:
Done